src/coreclr/jit/emitxarch.h

```diff
@@ -133,10 +133,8 @@ bool IsRedundantStackMov(instruction ins, insFormat fmt, emitAttr size, regNumbe
 static bool IsJccInstruction(instruction ins);
 static bool IsJmpInstruction(instruction ins);
 
-#ifdef TARGET_64BIT
```

Comment:
Is your TP impacted because you are extending it to x86?

Reply:
It would make sense if the TP was only impacted for FullOpts. But there is a TP impact for MinOpts which doesn't make sense because the optimizations only get kicked when optimizations are enabled. Maybe `IsRedundantMov` is spilling more than usual?